keyboards/keychron/k4/white/v2/rules.mk

```diff
@@ -0,0 +1,25 @@
+# project specific files
```

Comment:
Could this be converted to an info.json style instead (see keychron k8), as it seems that upstream is moving towards that format..

Reply:
I'm not as familiar with the info.json stuff, but I can dig into it and see.